plugins/by-name/dap-rr/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, ... }:
+let
+  inherit (lib) types mkOption;
+  inherit (lib.nixvim.options) pluginDefaultText;
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "dap-rr";
+  packPathName = "nvim-dap-rr";
+  package = "nvim-dap-rr";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  settingsOptions = {
+    mappings = mkOption {
+      # Without providing a `mappings` table, the `setup` function crashes with:
+      # bad argument #1 to 'pairs' (table expected, got nil)
+      default.__empty = null;
```

Comment:
Have you double checked that `__empty` doesn't get merged into normal attr definitions?

Normally option-default prio definitions _aren't_ merged with normal prio definitions, but I can't recall whether `attrsOf` handles that differently.

If the `__empty` _is_ merged into the final value then `toLuaObject` will always print `"{}"` regardless of what other attrs are in the value.